EpsilonDriverControls/make.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+if [ "$1" = "clean" ]
+then
+    (cd ../EpsilonDriverControls/ && make clean)
```

Comment:
Wait, doesn't `cd ../EpsilonDriverControls` just take you to the same spot?

Reply:
Oh wait yea true.

Reply:
But then when you're making the source code why do you browse to the folder when you're already in it when running make.sh?